packages/flutter/lib/src/material/app_bar_theme.dart

```diff
@@ -83,6 +89,15 @@ class AppBarTheme with Diagnosticable {
   /// If null, the value is adapted to current [TargetPlatform].
   final bool centerTitle;
 
+  /// This flag is deprecated and caps the text scaling to a maximum for the
```

Comment:
You should specify somehow that when this flag goes away the behavior will be to cap the title text scaling by default.

Reply:
I can add:
'This flag is going to be removed, to have text scaling capped by default.'

Reply:
Done.